source/common/stats/symbol_table_impl.cc

```diff
@@ -516,7 +526,9 @@ void StatNameStorageSet::free(SymbolTable& symbol_table) {
 SymbolTable::StoragePtr SymbolTableImpl::join(const StatNameVec& stat_names) const {
   uint64_t num_bytes = 0;
   for (StatName stat_name : stat_names) {
-    num_bytes += stat_name.dataSize();
+    if (!stat_name.empty()) {
```

Comment:
why do we need this now?

Reply:
There are places in the code where elements being join()ed are empty, and we have been skipping these.

Here though we just have another case of an empty-constructed StatName() will crash now if we don't special-case this.